src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * deprecated the "choices_as_values" option of ChoiceType
+ * deprecated support for Traversable in `ResizeFormListener::preSubmit` method
```

Comment:
I would say: "deprecated support for data objects that implements both ``\Traversable`` and ``\ArrayAccess`` in ``ResizeFormListener::preSubmit`` method" or something like that, what do you think?